mPowerSDK/TasksAndSteps/APHActivityManager.m

```diff
@@ -219,14 +221,31 @@ - (ORKOrderedTask *)createCustomWalkingTask
     [instructionStep setText:NSLocalizedStringWithDefaultValue(@"APH_WALKING_DESCRIPTION", nil, APHLocaleBundle(), @"This activity measures your gait (walk) and balance, which can be affected by Parkinson disease.", @"Description of purpose of walking activity.")];
     [instructionStep setDetailText:NSLocalizedStringWithDefaultValue(@"APH_WALKING_CAUTION", nil, APHLocaleBundle(), @"Please do not continue if you cannot safely walk unassisted.", @"Warning regarding performing walking activity.")];
     
-    NSString  *titleFormat = NSLocalizedStringWithDefaultValue(@"APH_WALKING_STAND_STILL_TEXT_INSTRUCTION", nil, APHLocaleBundle(), @"Turn around and stand still for %@ seconds", @"Written instructions for the standing-still step of the walking activity, to be filled in with the number of seconds to stand still.");
-    NSString  *titleString = [NSString stringWithFormat:titleFormat, APHLocalizedStringFromNumber(@(kStandStillDuration))];
-    NSString  *spokenInstructionFormat = NSLocalizedStringWithDefaultValue(@"APH_WALKING_STAND_STILL_SPOKEN_INSTRUCTION", nil, APHLocaleBundle(), @"Turn around and stand still for %@ seconds", @"Spoken instructions for the standing-still step of the walking activity, to be filled in with the number of seconds to stand still.");
-    NSString  *spokenInstructionString = [NSString stringWithFormat:spokenInstructionFormat, APHLocalizedStringFromNumber(@(kStandStillDuration))];
+    ORKInstructionStep *instructionStep2 = (ORKInstructionStep *)orkTask.steps[1];
+    [instructionStep2 setText:NSLocalizedStringWithDefaultValue(@"APH_WALKING_DESCRIPTION_2", nil, APHLocaleBundle(), @"\u2022 Please wear a comfortable pair of walking shoes and find a flat, smooth surface for walking.\n\n\u2022 Try to walk continuously by turning at the ends of your path, as if you are walking around a cone.\n\n\u2022 Importantly, walk at your normal pace. You do not need to walk faster than usual.", @"Detailed instructions for performing walking activity.")];
+    [instructionStep2 setDetailText:NSLocalizedStringWithDefaultValue(@"APH_WALKING_DESCRIPTION_2_DETAIL", nil, APHLocaleBundle(), @"Put your phone in a pocket or bag and follow the audio instructions.", @"Instructions for how to proceed to active step of walking activity.")];
     
-    ORKActiveStep *activeStep = (ORKActiveStep *)orkTask.steps[5];
-    [activeStep setTitle:titleString];
-    [activeStep setSpokenInstruction:spokenInstructionString];
+    NSString  *titleFormatWalk = NSLocalizedStringWithDefaultValue(@"APH_WALKING_WALK_TEXT_INSTRUCTION", nil, APHLocaleBundle(), @"Walk back and forth for %@ seconds.", @"Written instructions for the walking step of the walking activity, to be filled in with the number of seconds to walk.");
```

Comment:
Can't think of a language where it wouldn't, in fact AFAIK this is the recommended way to localize strings with numbers in, but yeah, can edit later if this turns out not to be the case.